packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -21,29 +21,77 @@
         :name="shareTypeKey"
         class="files-collaborators-collaborator-indicator"
       />
-      <div class="oc-text-truncate">
-        <p v-oc-tooltip="shareDisplayNameTooltip" class="oc-text-bold oc-text-truncate oc-m-rm">
-          <span
-            aria-hidden="true"
-            class="files-collaborators-collaborator-name"
-            v-text="shareDisplayName"
-          />
-          <span
-            v-if="shareAdditionalInfo"
-            aria-hidden="true"
-            class="files-collaborators-collaborator-additional-info"
-            v-text="shareAdditionalInfo"
-          />
+      <div class="oc-text-truncate oc-width-1-1">
+        <div class="oc-flex oc-flex-middle">
+          <span v-oc-tooltip="shareDisplayNameTooltip" class="oc-text-truncate oc-m-rm">
+            <span
+              aria-hidden="true"
+              class="files-collaborators-collaborator-name"
+              v-text="shareDisplayName"
+            />
+          </span>
           <span class="oc-invisible-sr" v-text="screenreaderShareDisplayName" />
-        </p>
-        <p class="oc-m-rm oc-flex">
-          <span
-            aria-hidden="true"
-            class="files-collaborators-collaborator-share-type"
-            v-text="shareTypeText"
-          />
-          <span v-if="sharedParentRoute" class="oc-resource-indicators oc-text-truncate">
-            <span class="oc-mx-s">·</span>
+          <oc-button
+            :id="`share-access-details-toggle-${share.id}`"
+            class="oc-ml-xs files-collaborators-collaborator-access-details-button"
+            appearance="raw"
+          >
+            <oc-icon name="information" fill-type="line" size="small" />
+          </oc-button>
+          <oc-drop :toggle="`#share-access-details-toggle-${share.id}`" mode="click">
+            <p v-translate class="oc-text-bold">Access details</p>
+            <oc-list>
+              <li class="oc-flex">
```

Comment:
`li` needs to be removed completely if there is no additional info (e.g. for group shares this is the case) => add a `v-if` and check if the additional info isn't empty.